LICENSE

```diff
@@ -0,0 +1,339 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
If you have no objection, I would prefer to go with GPLv3 instead.

Reply:
No problem - it's mostly to put a license in there, as the EU is quite picky with this